### DISCUSSION TITLE
Observer Condition - Distance [Shader Graph Fade In/Out]

### DISCUSSION THREAD

**Discussion by iMlearnDinG** [FirstGearGames/FishNet#601] [Feature Request]
Greetings Team,

I would like to request a simple shader graph for fading objects in and out when they trigger observer condition thresholds for a more smoother object transition. 

Kind Regards,

**Comment by FirstGearGames** [comment]
Hey! You might want to see if someone in the Discord can do this. That would be outside the scope of FN features.

BUT, you might be able to make your own object pool (by inheriting our ObjectPool in FishNet) and hold off disabling the objects and hide the renderers overtime instead.

https://fish-networking.gitbook.io/docs/manual/guides/spawning/object-pooling#general
